### DISCUSSION TITLE
'Custom-work-for-brackets' plugin for Phoenix Code

### DISCUSSION THREAD

**Discussion by Unlicensed-ZZZ** [phcode-dev/phoenix#1678] [Ideas]
Dear team!
There is the extremely useful plugin for Brackets 'custom-work-for-brackets', can be found here: https://github.com/alessandrio/custom-work-for-brackets. It also works in Phoenix Code too, but with a number of distortions (for example: unneeded extra lines in work plane, don't move slide-bar and its icons).
A huge request to adapt and include it in Phoenix Code plugins or (may be better) add similar functionality to the product.

**Comment by abose** [comment]
Thanks for the note. The extension was removed from the store as it broke down a few things if i remember correctly. 
I too agree that some of the extension features needs to be default in phoenix code. 

These are tracked as part of: https://github.com/orgs/phcode-dev/projects/9/views/1

* https://github.com/phcode-dev/phoenix/issues/161
